netci.groovy

```diff
@@ -257,18 +257,35 @@ if (!branch.endsWith('-ci')) {
 CreateStyleCheckTasks('./jenkins/check_eol.sh', 'ubuntu_check_eol', 'EOL Check')
 CreateStyleCheckTasks('./jenkins/check_copyright.sh', 'ubuntu_check_copyright', 'Copyright Check')
 
+// --------------
+// XPLAT BRANCHES
+// --------------
+
+def isXPlatCompatibleBranch = branch.startsWith('master') || branch.startsWith('linux') || branch.startsWith('xplat') ||
+    (branch.startsWith('release') && !(branch =~ /^release\/1\.[12](\D.*)?$/))
+
+// Include these explicitly-named branches
+def isXPlatDailyBranch = branch in ['master', 'linux', 'xplat']
+// Include some release/* branches
+if (branch.startsWith('release')) {
+    // Allows all future release/* branches which should be considered for daily builds of XPlat configs.
+    // RegEx matches e.g. release/1.1, release/1.2, release/1.3-ci, but not e.g. release/2.0, release/1.3, or release/1.10
+    releaseExclusion = branch =~ /^release\/((1\.[12](\D.*)?)|(\d+\.\d+\D.*))$/
+    isXPlatDailyBranch |= !releaseExclusion
+}
+
 // -----------------
 // LINUX BUILD TASKS
 // -----------------
 
-if (branch.startsWith('linux') || branch.startsWith('master')) {
+if (isXPlatCompatibleBranch) {
```

Comment:
We do have incompatible branches: `release/1.1` and `release/1.2` -- builds from those branches can't build XPlat or run tests. The fancy RegEx I'm using here is primarily so that the theoretical 1.10 which comes after 1.1 won't be considered as "incompatible". We don't yet know when/whether we will version bump to 2.0, so we could in theory reach v1.10.